packages/@uppy/compressor/src/index.js

```diff
@@ -52,15 +52,22 @@ export default class Compressor extends BasePlugin {
           this.uppy.log(`[Image Compressor] Image ${file.id} compressed by ${prettierBytes(compressedSavingsSize)}`)
           totalCompressedSize += compressedSavingsSize
           const { name, type, size } = compressedBlob
-          const extension = name && getFileNameAndExtension(name).extension
+          const { extension } = name && getFileNameAndExtension(name)
```

Comment:
What are the possible values for `name`? Can it be nullish? If so, we cannot use this syntax, or you'll get `Cannot use destructuring on undefined` or something like that.

Reply:
I think it shouldn't be nullish, but changed the syntax regardless. It's not bulletproof though.